app/components/graph-editor/defaults.ts

```diff
@@ -99,20 +97,17 @@ export const NUDGE: number = 3;
 export const STICKY_DELAY: number = 500;
 
 /**
- * NODE_DEFAULTS  
+ * NODE_PROPERTIES  
  *   Default property values for drawable nodes.
- * 
- *   Only `DrawableNode` properties should be in this object and nothing else.
- *   This object is used as the template for `isDrawableNode`.
- */
-export const NODE_DEFAULTS: DrawableNode = {
-    position: { x: 0, y: 0 },
-    label: "",
-    shape: "circle",
-    color: "#fff200",
-    borderColor: "#000",
-    borderStyle: "solid",
-    borderWidth: 1
+ */
+export const NODE_PROPERTIES = {
```

Comment:
This probably won't happen until after Beta, but I imagine the settings window could have some behavior where each component that wants settings can have their say. So the settings window would have any number of sections, and the IDE could have what it wants, the graph editor could have what it wants, the plugin service could have what it wants, etc...